podcasts/EpisodesDataManager.swift

```diff
@@ -0,0 +1,127 @@
+import PocketCastsDataModel
+import PocketCastsServer
+import DifferenceKit
+
+/// Returns a list of episodes for an specific section
+/// Depending on the section, it returns a DifferenceKit ArraySection
+class EpisodesDataManager {
```

Comment:
I'm not 100% sold on the name of this class. Any suggestions are highly appreciated. Maybe `EpisodesListManager` is a better choice?

Reply:
Scanning through this briefly, it looks more like a Helper class (as opposed to a Manager class)? I see it also calls out to several other Helpers.

Reply:
I agree, the naming of this isn't great since we also have the `EpisodeDataManager`. I think maybe using something like 'EpisodesHelper'?

Reply:
I agree! It's ok if I change this in a follow-up PR? Just want to avoid a few merging conflicts.

Reply:
Sounds good.